### DISCUSSION TITLE
Can someone point me to a quick Github Actions that works?

### DISCUSSION THREAD

**Discussion by localhost-labs** [community/community#66070] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I just need a sample

**Comment by chrisreeves-** [comment]
There is a sample here: https://docs.github.com/en/actions/quickstart

**Comment by queenofcorgis** [comment]
GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
